src/components/RightBoxSection.ts

```diff
@@ -1,26 +1,24 @@
 import { CSSProperties, ReactNode } from 'react';
 import { div, h, h3 } from 'react-hyperscript-helpers';
 import Collapse from 'src/components/Collapse';
-import { Ajax } from 'src/libs/ajax';
 import colors from 'src/libs/colors';
-import Events, { extractWorkspaceDetails } from 'src/libs/events';
 import * as Style from 'src/libs/style';
 import { useLocalPref } from 'src/libs/useLocalPref';
-import { WorkspaceWrapper } from 'src/workspaces/utils';
 
-interface RightBoxSectionProps {
+export type RightBoxSectionProps = {
```

Comment:
Out of curiosity, why did you choose a type over an interface? I don't think it matters either way in this case, but I'm still trying to learn when to use one vs. the other.

Reply:
Yeah, I too am still trying to figure that out myself. Personally, I try to use interface to describe objects that come from some external source, like an API, or anything that has a low threshold for being updated. I usually use types for things that I define myself, like small components that may be re-used in the future and need additional props added/removed. I'm still trying to learn so my understanding continues to evolve as I learn more.